apps/functions-subscription/src/main.ts

```diff
@@ -87,3 +94,14 @@ if (config.subscriptionRequest.consumer === 'on')
     cardinality: 'many',
     handler: makeSubscriptionRequestConsumerHandler(env),
   });
+
+if (config.activations.consumer === 'on') {
+  app.cosmosDB('activationConsumer', {
+    connection: 'ActivationConsumerCosmosDBConnection',
+    databaseName: config.cosmosdb.databaseName,
+    containerName: 'activations',
+    leaseContainerName: config.cosmosdb.leasesContainerName,
+    leaseContainerPrefix: 'activations-',
```

Comment:
**note**: the `-` suffix is used to separate the name in the `leases` container.

without it, we'd have an id like `activations<something-else>`; with the `-` we are having `activations-<something-else>`.